docs/static/releasenotes.asciidoc

```diff
@@ -36,6 +37,46 @@ This section summarizes the changes in the following releases:
 * <<logstash-7-0-0-alpha2,Logstash 7.0.0-alpha2>>
 * <<logstash-7-0-0-alpha1,Logstash 7.0.0-alpha1>>
 
+[[logstash-7-10-1]]
+=== Logstash 7.10.1 Release Notes
+
+==== Notable issues fixed
+
+===== Fix docker image metadata
+Removed non-OCI compliant freeform metadata labels, started using consistent build date for all the docker images and dockerfiles. https://github.com/elastic/logstash/pull/12451[#12451]
+Reintroduces the labels for the ubi8 image only. https://github.com/elastic/logstash/pull/12500[#12500]
+
+===== Exclude file with non-commercial license
+Bundled did_you_mean gem had a CC NonCommercial license in an included file, these files has been removed. https://github.com/elastic/logstash/pull/12460[#12460]
+
+===== Support recreation of same pipeline through centralized pipeline management
+When users attempted to delete and recreate a pipeline with the same identifier and configuration, Logstash was unable to pick up the new pipeline. https://github.com/elastic/logstash/issues/12387[#12387]
+
+==== Plugins
+
+*Azure_event_hubs Input - 1.2.3*
+
+* Fixed missing configuration of `prefetch_count` and `receive_timeout` https://github.com/logstash-plugins/logstash-input-azure_event_hubs/pull/61[#61]
+
+*Http Input - 3.3.6*
+
+* Fixes a regression introduced in **3.1.0**'s migration to the Netty back-end that broke some users'
+   browser-based workflows. When a plugin configured to require Basic authentication receives a request that does not
+   include authentication, it now appropriately includes an `WWW-Authenticate` header in its `401 Unauthorized` response,
+   allowing the browser to collect credentials before retrying the request.
+
+*Sqs Input - 3.1.3*
+
+* Fix: retry networking errors (with backoff) https://github.com/logstash-plugins/logstash-input-sqs/pull/57[#57]
+
+*Kafka Integration - 10.5.3*
+
+* Fix: set (optional) truststore when endpoint id check disabled https://github.com/logstash-plugins/logstash-integration-kafka/pull/60[#60]. +
+  Since **10.1.0** disabling server host-name verification (`ssl_endpoint_identification_algorithm => ""`) did not allow the (output) plugin to set `ssl_truststore_location => "..."`.
+
+* Docs: explain group_id in case of multiple inputs https://github.com/logstash-plugins/logstash-integration-kafka/pull/59[#59]
```

Comment:
```suggestion
* Fix: set (optional) truststore when endpoint id check disabled https://github.com/logstash-plugins/logstash-integration-kafka/pull/60[#60]. +
  Since **10.1.0** disabling server host-name verification (`ssl_endpoint_identification_algorithm => ""`) did not allow the (output) plugin to set `ssl_truststore_location => "..."`.
* Docs: explain group_id in case of multiple inputs https://github.com/logstash-plugins/logstash-integration-kafka/pull/59[#59]
```

Reply:
I don't any diff in this change :-(, is it the `+` sign at the end of line 74 ?

Reply:
Line 73 is deleted (empty bullet point)